src/cmd/manage_nsfs.js

```diff
@@ -774,10 +774,10 @@ async function validate_account_args(data, action) {
 
         if (is_undefined(data.access_keys[0].secret_key)) throw_cli_error(ManageCLIError.MissingAccountSecretKeyFlag);
         if (is_undefined(data.access_keys[0].access_key)) throw_cli_error(ManageCLIError.MissingAccountAccessKeyFlag);
-        if (data.nsfs_account_config.gid && (is_undefined(data.nsfs_account_config.uid))) {
```

Comment:
In addition to your change, I think either the function name `is_undefined` or the implementation of the function should be changed. 
- Function name change - The current implementation doesn't check if the value is undefined but if the value is Falsy (here is a [link](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)), and 0 is Falsy.
- Change function implementation - to check if the value is_undefined explicitly (includes the SensitiveString case).
I would go on the second option.